File: video-embeddings/retrieve-status.js
import { TwelveLabs } from 'twelvelabs-js';

(async () => {
  const client = new TwelveLabs({ apiKey: process.env.API_KEY});

  const task = await client.embed.task.status('66ed21d25bfc453560996948')
  
  console.log(`Task ID: ${task.id}`);
  console.log(`Engine Name: ${task.engineName}`);
  console.log(`Status: ${task.status}`);
})();

File: video-embeddings/retrieve-status.py
import os
from twelvelabs import TwelveLabs

client = TwelveLabs(api_key=os.environ.get("API_KEY"))

task = client.embed.task.status(task_id="66ed9d64808d95368f6f8bd3")

print(f"Task ID: {task.id}")
print(f"Engine Name: {task.engine_name}")
print(f"Status: {task.status}")
